Gruntfile.js

```diff
@@ -351,6 +351,15 @@ module.exports = function (grunt) {
             test : {
                 files: ['<%= meta.test %>', 'test/**/*'],
                 tasks: ['eslint:test']
+            },
+            bramble : {
+                files: [
+                    'src/bramble/client/**/*.js',
+                    'src/bramble/thirdparty/**/*.js',
+                    'src/bramble/ChannelUtils.js',
+                    'thirdparty/filer/dist/filer.min.js'
```

Comment:
let's remove this and simplify the `files` to be just `['src/bramble/**/*']`

Reply:
@gideonthomas The bramble.js is built from https://github.com/mozilla/brackets/blob/master/Gruntfile.js#L228-L234
That's why I want to all these files. Do you want to change to `['src/bramble/**/*']`

Reply:
yes please. We do not want to watch files in `thirdparty` because they will (should) never change.